app/components/App.js

```diff
@@ -2,6 +2,7 @@ import React, { Component } from 'react';
 import Navigation from './Navigation';
 import CategoryPage from './Find/FindPage';
 import ResourcesTable from './Search/ResourcesTable';
+import 'react-select/dist/react-select.css';
```

Comment:
It looks like this works fine. I found this thread that shows how you can do it with the CSS modules-loader, but we're not using it: https://github.com/JedWatson/react-select/issues/1324#issuecomment-256591621